repos/system_upgrade/common/libraries/rhsm.py

```diff
@@ -169,13 +168,6 @@ def get_available_repo_ids(context):
         )
 
     repofiles = repofileutils.get_parsed_repofiles(context)
```

Comment:
this call leads to the situation when all discovered repositories have the `custom` kind set, so repositories inside `RHSMInfo` msg are marked as `custom`. Discussed with @vinzenz , he will check whether the logic of selecting (custom, rhsm, rhui) kind could be moved into the library instead of having it in the private library of the actor.